### ISSUE TITLE
poetry-setup==0.3.5 broken on PyPI

### ISSUE THREAD

**Issue by rooterkyberian** [orsinium/poetry-setup#5]
None of the templates seems to be included in the package.

```
Traceback (most recent call last):
  File "/home/user/venv/bin/poetry-setup", line 9, in <module>
    load_entry_point('poetry-setup==0.3.5', 'console_scripts', 'poetry-setup')()
  File "/home/user/venv/lib/python3.6/site-packages/poetry_setup/core.py", line 111, in main
    PoetrySetup(*argv).sync()
  File "/home/user/venv/lib/python3.6/site-packages/poetry_setup/core.py", line 92, in sync
    document = self.get_requirements(optional=False)
  File "/home/user/venv/lib/python3.6/site-packages/poetry_setup/core.py", line 56, in get_requirements
    with self.requirements_path.open(encoding='utf-8') as f:
  File "/usr/lib/python3.6/pathlib.py", line 1183, in open
    opener=self._opener)
  File "/usr/lib/python3.6/pathlib.py", line 1037, in _opener
    return self._accessor.open(self, flags, mode)
  File "/usr/lib/python3.6/pathlib.py", line 387, in wrapped
    return strfunc(str(pathobj), *args)
FileNotFoundError: [Errno 2] No such file or directory: '/home/user/venv/lib/python3.6/site-packages/poetry_setup/templates/requirements.txt.j2'
```

**Comment by rooterkyberian** [comment]
Seems to be reoccurring problem #1

**Comment by tkschmidt** [comment]
same issue for version 0.3.4
```
Writing lock file        

  - Installing wheel (0.32.2)                      
  - Updating distributed (1.24.1 -> 1.24.2)        
  - Updating poetry-setup (0.3.5 -> 0.3.4)         
  - Removing locket (0.2.0)                        
  - Removing partd (0.3.9)                         
(venv) 13:50:50 X@Y ~/code/msp_conv $ poetry-setup                                        
Traceback (most recent call last):                 
  File "/home/X/code/msp_conv/venv/bin/poetry-setup", line 11, in <module>                     
    sys.exit(main())     
  File "/home/X/code/msp_conv/venv/lib/python3.6/site-packages/poetry_setup/core.py", line 108, in main
    PoetrySetup(*argv).sync()                      
  File "/home/X/code/msp_conv/venv/lib/python3.6/site-packages/poetry_setup/core.py", line 31, in __init__
    self.package = self._get_package(path)         
  File "/home/X/code/msp_conv/venv/lib/python3.6/site-packages/poetry_setup/core.py", line 38, in _get_package
    builder = Builder(poetry, venv=None, io=NullIO)                                                   
TypeError: __init__() got an unexpected keyword argument 'venv' 
```

**Comment by orsinium** [comment]
@rooterkyberian, thank you for your report. It was old setuptools on my machine. Fixed.

**Comment by orsinium** [comment]
@tkschmidt, it was fixed in 0.3.5. Please, update poetry-setup to current 0.3.6 verison:

```
pip3 install --no-cache-dir -U poetry-setup
```
